src/main/java/me/tsukanov/counter/ui/CounterFragment.java

```diff
@@ -94,13 +94,27 @@ public void onClick(View v) {
                 increment();
             }
         });
+       incrementButton.setOnLongClickListener(new View.OnLongClickListener() {
+
+            public boolean onLongClick(View v) {
+                incrementDecade();
```

Comment:
Would it make more sense to refactor `increment()` and related methods to allow passing delta value instead of having it as 1?

Reply:
I tried to make it as simple as possible as I am new to Android Development.

Reply:
Fair enough.